doc/md/serving/intro.md

```diff
@@ -0,0 +1,38 @@
+---
+id: atlas-serving
+slug: /serving/intro
+title: Serving
+---
+
+## Introduction
+
+Usually as a team using Atlas, you would like to deploy it as a service in your cloud data center and enjoy a centralized 
+instance.
+When using Atlas in such cases we recommend using persistent storage.
+Atlas encrypts all sensitive information using [AEAD](https://developers.google.com/tink/aead?hl=en) encryption with
```

Comment:
Atlas uses [tink](https://developers.google.com/tink), a battle-tested encryption library created at Google, to encrypt all sensitive information. Following recommendations from the developers of tink, Atlas uses [AEAD](https://developers.google.com/tink/aead?hl=en) encryption with an *AES128_GCM* type key.